libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -52,6 +67,54 @@ void AC_Avoid::adjust_velocity(float kP, float accel_cmss, Vector3f &desired_vel
     desired_vel.y = des_vel_xy.y;
 }
 
+// adjust roll-pitch to push vehicle away from objects
+// roll and pitch value are in centi-degrees
+void AC_Avoid::adjust_roll_pitch(float &roll, float &pitch, float angle_max)
```

Comment:
as roll and pitch are not changed it could be good to pass them by const reference